src/commands/edit-input-schema.js

```diff
@@ -9,19 +9,32 @@ const { cryptoRandomObjectId } = require('@apify/utilities');
 
 const { ApifyCommand } = require('../lib/apify_command');
 const outputs = require('../lib/outputs');
+const { readInputSchema, DEFAULT_INPUT_SCHEMA_PATHS } = require('../lib/input_schema');
 
 const INPUT_SCHEMA_EDITOR_BASE_URL = 'https://apify.github.io/input-schema-editor-react/';
 const INPUT_SCHEMA_EDITOR_ORIGIN = new URL(INPUT_SCHEMA_EDITOR_BASE_URL).origin;
 
-const DEFAULT_INPUT_SCHEMA_PATH = './INPUT_SCHEMA.json';
-
 // Not really checked right now, but it might come useful if we ever need to do some breaking changes
 const API_VERSION = 'v1';
 
 class EditInputSchemaCommand extends ApifyCommand {
     async run() {
         const { args } = this.parse(EditInputSchemaCommand);
-        const { path = DEFAULT_INPUT_SCHEMA_PATH } = args;
+        let path = null;
+
+        // This call fails if the input schema is invalid JSON
+        const inputSchemaWithPath = await readInputSchema(args.path);
+
+        if (!inputSchemaWithPath) {
+            outputs.warning('Input schema has not been found. Default one will be created.');
+            fs.writeFileSync(DEFAULT_INPUT_SCHEMA_PATHS[0], '{}');
+            path = DEFAULT_INPUT_SCHEMA_PATHS[0];
+        } else if (!inputSchemaWithPath.path) {
+            // If path is not returned in inputSchemaWithPath, it means the input schema must be directly embedded as object in actor.json
+            throw new Error('Cannot edit an input schema directly embedded in .actor/actor.json at this time. Please, submit a feature request!');
```

Comment:
Don't throw the error, it would show up as an unhandled error (I think). Just do `outputs.error(...); return false`, that will end up nicer.

Reply:
IT's exactly the same (first throwing,s second ouputs.error)
<img width="778" alt="CleanShot 2022-11-28 at 13 19 44@2x" src="https://user-images.githubusercontent.com/1821884/204276279-d132d066-395f-41d5-ae33-4876ee16a191.png">